src/components/NavItems/tools/SemanticSearch/components/ResultDisplayItem.jsx

```diff
@@ -58,7 +65,7 @@ const ResultDisplayItem = ({
               spacing={2}
             >
               <Stack direction="column">
-                <Typography>
+                <Typography align="left">
```

Comment:
Could you confirm this will work for arabic language? I think using `align="start"` would be better